yurt/orchestration/roles/db/tasks/main.yml

```diff
@@ -3,9 +3,9 @@
 - name: Install PostgreSQL
   apt: name={{ item }} update_cache={{ update_apt_cache }} state=installed
   with_items:
-    - postgresql=9.4*
-    - postgresql-contrib=9.4*
-    - python-psycopg2=2.5.*
+    - postgresql
```

Comment:
Can you put variables in the `with_items`? Would be cool if you could pin these dependencies if you could grab the variable from a setting.